modules/installation-user-infra-machines-advanced.adoc

```diff
@@ -277,71 +277,3 @@ For PXE or ISO boots, you can create the Ignition config
 and `APPEND` the `ignition.config.url=` option to identify the location of
 the Ignition config. You also need to append `ignition.firstboot ignition.platform.id=metal`
 or the `ignition.config.url` option will be ignored.
-
```

Comment:
The preceding paragraph about the "live install Ignition config" should be updated.  Instead of saying "must be created manually" and "not supported", we can point users to the customize commands.